src/components/MapView/index.tsx

```diff
@@ -212,8 +227,43 @@ function MapView({ classes }: MapViewProps) {
         containerStyle={{
           height: '100%',
         }}
-        onClick={() => {
+        onClick={(map: Map, evt: any) => {
           dispatch(hidePopup());
+          // Get layers that have getFeatureInfo option.
+          const featureInfoLayers = getActiveFeatureInfoLayers(map);
+          if (featureInfoLayers.length === 0) {
+            return;
+          }
+
+          const dateFromRef = (selectedDateRef.current?.props.selected as Date)
+            .toISOString()
+            .split('T')[0];
+
+          const params = getFeatureInfoParams(map, evt, dateFromRef);
+          makeFeatureInfoRequest(featureInfoLayers, params).then(
+            (result: any) => {
+              if (result === null) {
```

Comment:
falsey check instead just in case the value is undefined?

Reply:
Included in 30953ddd1dd6300a0fc259ec913e13a3b75fd2ba